cnf-certification-test/platform/nodetainted/nodetainted_test.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 /*func Test_NewNodeTaintedTester(t *testing.T) {
```

Comment:
This commented section could just be removed now.

Reply:
fixed